src/transaction/declare_v2.rs

```diff
@@ -25,7 +26,8 @@ use num_traits::Zero;
 use starknet_contract_class::EntryPointType;
 use std::collections::HashMap;
 
-use super::Transaction;
+/// Represents a declare transaction in the starknet network.
```

Comment:
here you can specify that DeclareV2 is meant to be used to declare Cairo contracts using the new syntax.

Reply:
fixed